marathon/hbase-regionerver.json

```diff
@@ -1,14 +1,19 @@
 {
     "id": "/cell-os/hbase-1/regionservers", 
-    "cmd": "su hadoop -c \"curl -vvv -o /hbase/conf/hdfs-site.xml http://HDFS_NAMENODE:50070/conf && /hbase/bin/hbase regionserver -D hbase.regionserver.port=$PORT0 -Dhbase.regionserver.info.port=$PORT1 -Dhbase.rootdir='hdfs://HDFS_NAMEPACE_ID/hbase-1' -Dhbase.zookeeper.quorum=ZK_QUORUM -D zookeeper.znode.parent=/hbase-1 start\"",
+    "cmd": "su hadoop -c \"/entrypoint.sh && curl -vvv -o /hbase/conf/hdfs-site.xml http://HDFS_NAMENODE_HOST/conf && /hbase/bin/hbase regionserver -D hbase.regionserver.port=$PORT0 -Dhbase.regionserver.info.port=$PORT1 -Dhbase.rootdir='hdfs://HDFS_NAMESPACE_ID/hbase-1' -Dhbase.zookeeper.quorum=ZK_QUORUM -D zookeeper.znode.parent=/hbase-1 start\"",
  
     "container": {
         "docker": {
-            "image": "cellos/hbase:1.0.0"
+            "image": "cellos/hbase"
```

Comment:
This will just bring latest, but that can be problematic as, unless we have the actual hbase version in the container name, and use the versioning tags (e.g. latest) for the container version you'll get whatever container was published lasts (0.98, 1.x etc.) Instead we should probably have :`1.2.0-container version`